src/ConnectionAutomaton.cpp

```diff
@@ -320,13 +320,13 @@ void ConnectionAutomaton::onConnectionFrame(
   switch (type) {
     case FrameType::KEEPALIVE: {
       Frame_KEEPALIVE frame;
-      if (!deserializeFrameOrError(frame, std::move(payload))) {
+      if (!deserializeFrameOrError(isResumable_, frame, std::move(payload))) {
```

Comment:
Note I started implementing looking at frame size but it gets tricky fast as the KEEPALIVE frame has variable data.